data/styles/vehicle/include/priorities_4_overlays.prio.txt

```diff
@@ -701,7 +701,7 @@ amenity-fast_food                                   # icon z17- (also has captio
 
 amenity-bank                                        # icon z17- (also has caption(optional) z17-)
 amenity-parcel_locker                               # icon z17- (also has caption(optional) z17-)
-amenity-police                                      # icon z17-
+amenity-police                                      # icon z17- (also has area z12-)
```

Comment:
I think the area fill is not needed / not useful for a driver.
E.g. we don't use area fills for unis, hospitals...